modules/efk-logging-elasticsearch-exposing.adoc

```diff
@@ -40,28 +62,67 @@ metadata:
   name: elasticsearch
   namespace: openshift-logging
 spec:
-  host:
+  host: 
   to:
     kind: Service
     name: elasticsearch
   tls:
-    termination: reencrypt <1>
-    destinationCACertificate: |-   <2>
-      -----BEGIN CERTIFICATE-----
-      [...]
-      -----END CERTIFICATE-----
+    termination: reencrypt
+    destinationCACertificate: <1>
 ----
-<1> Set the termination field to reencrypt.
-<2> Add the contents of the *_admin-ca_* file.
+<1> Add the Elasticsearch CA ceritifcate or use the command in the next step. You do not need to set the `spec.tls.key`, `spec.tls.certificate` and `spec.tls.caCertificate` parameters
+required by some reencrypt routes.
 
-. Use the following command to create the service:
+.. Run the following commnd to add the Elasticsearch CA certificate to the route YAML you created:
 +
 ----
-$ oc create -f my_es_route.yaml
+$cat ./admin-ca | sed -e "s/^/      /" >> <file-name>.yaml
+----  
+
+.. Run the following command to create the route:
++
+----
+$oc create -f <file-name>.yaml
+
+route.route.openshift.io/elasticsearch created
 ----
++
+For an example reencrypt route object, see Re-encryption Termination.
++
+//This line ^^ will be linked when the topic is available.
+
+. Check that the Elasticsearch service is exposed:
 
-. Check that the ElasticSearch service is exposed:
+.. Get the token of this ServiceAccount to be used in the request:
 +
 ----
-curl --silent --insecure -H "Authorization: Bearer $( oc whoami -t )" "https://$( oc get route elasticsearch -o jsonpath='{.spec.host}' ):443/.operations.*/_search" | jq
+$ token=$(oc whoami -t)
 ----
+
+.. Run a command similar to the following to access Elasticsearch through the exposed route:
++
+----
+curl -tlsv1.2 -v --insecure -H "Authorization: Bearer me3IL_WmD_I_McBUm2uhxIayUKped-H3L1njlRRxPlE" "https://elasticsearch-openshift-logging.apps.ntest10.devcluster.openshift.com/.operations.*/_search?size=1" | jq
```

Comment:
@nhosoi Is this cURL command right? I run it and get 
`parse error: Invalid numeric literal at line 2, column 0`

Reply:
And, I need to replace `ntest10.devcluster` with my cluster name, in my case: `mburke13.devcluster.`?

Reply:
Could you please do
```
token=$(oc whoami -t)
elasticsearch_fqdn=$(oc get route elasticsearch -o jsonpath='{.spec.host}')
```
then run this?
`curl -tlsv1.2 --insecure -H "Authorization: Bearer $token" "https://$elasticsearch_fqdn/.operations.*/_search?size=1"`
what do you get?

Reply:
I get raw HTML output that includes: `Application is not available`

Reply:
hmmm, could you give me these outputs?
`echo $elasticsearch_fqdn`
and
`oc exec $YOUR_ES_POD_NAME -- curl -s -k --cert /etc/elasticsearch/secret/admin-cert --key /etc/elasticsearch/secret/admin-key "https://localhost:9200/.operations.*/_search?size=1"`

Reply:
@nhosoi A clue:
```
oc get routes
NAME            HOST/PORT                                                         PATH   SERVICES        PORT    TERMINATION          WILDCARD
elasticsearch   ExtendedValidationFailed 
```

Reply:
echo $elasticsearch_fqdn
elasticsearch-openshift-logging.apps.mburke13.devcluster.openshift.com

Reply:
Should this be oc exec $ESPOD?
```
oc exec $ESPOD -- curl -s -k --cert /etc/elasticsearch/secret/admin-cert --key /etc/elasticsearch/secret/admin-key "https://localhost:9200/.operations.*/_search?size=1"
Defaulting container name to elasticsearch.
Use 'oc describe pod/elasticsearch-clientdatamaster-0-1-7578f9bd75-9xxhl -n openshift-logging' to see all of the containers in this pod.
exec failed: container_linux.go:336: starting container process caused "exec: \"elasticsearch-clientdatamaster-0-2-85d99979b6-c6pfw\": executable file not found in $PATH"
command terminated with exit code 1

```

Reply:
Sorry, I meant $YOUR_ES_POD_NAME was going to be replaced with your elasticsearch pod name.  (I guess you already figured it out.)  But I've never seen the exec failed error...
Is elasticsearch-clientdatamaster-0-1-7578f9bd75-9xxhl your elasticsearch pod name?
What do you get from "oc get pods"?